api/gateway/share.go

```diff
@@ -20,7 +19,7 @@ const (
 	namespacedDataEndpoint   = "/namespaced_data"
 )
 
-var nIDKey = "nid"
```

Comment:
Forgot to change the actual string and it's breaking....

Don't have processing power rn to assess whether this worth a breakage here or not

Reply:
We are not deprecating this, so it might be worth it. Does it need to changes somewhere outside of go-cnc?